### ISSUE TITLE
What about report or alert that will identify products about to expire among many other products

### ISSUE THREAD

**Issue by trendcnet** [jekkos/opensourcepos#510]

**Comment by RamkrishnaMondal** [comment]
Yes would be very useful for items like medicine. But need help from more and more developers. If you are one please contribute by adding features.

**Comment by odiea** [comment]
Isn't that covered with Low Inventory Report? It currently shows low Inventory when the Items remaining are at or below the Reorder Level.

**Comment by arionkosturi** [comment]
He's talking about expiration date.
You could use custom fields to input the expiration date

**Comment by odiea** [comment]
Thank you. I never considered that as an option.
